### PR TITLE
Add Albis-Elcon BIG 4862

### PR DESCRIPTION
**Pull request by c-po**
Device used e.g. by German Telecom for their EC2.0 circuits

### CODE REVIEW

File: device-types/Albis-Elcon/BIG2862.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+manufacturer: Albis-Elcon
+model: BIG2862
+slug: big2862
```
Comment (danner26) [inline device-types/Albis-Elcon/BIG2862.yaml:4]: please append `albis-elcon-` to the slug

File: device-types/Albis-Elcon/BIG4862.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
+---
+manufacturer: Albis-Elcon
+model: BIG4862
+slug: big4862
```
Comment (danner26) [inline device-types/Albis-Elcon/BIG4862.yaml:4]: please append `albis-elcon-` to the slug

**Review by danner26 — CHANGES_REQUESTED** [review]
Everything looks good, please just fix the slugs

File: device-types/Albis-Elcon/BIG2862.yaml (hunk 1 above)
Comment (c-po) [reply device-types/Albis-Elcon/BIG2862.yaml]: done

File: device-types/Albis-Elcon/BIG4862.yaml (hunk 2 above)
Comment (c-po) [reply device-types/Albis-Elcon/BIG4862.yaml]: done

**Review by danner26 — APPROVED** [review]

**Pull request merged**